src/crypto/crypto_context.cc

```diff
@@ -1299,7 +1301,7 @@ void SecureContext::GetCertificate(const FunctionCallbackInfo<Value>& args) {
 
 namespace {
 unsigned long AddCertsFromFile(  // NOLINT(runtime/int)
-    X509_STORE* store,
```

Comment:
The original code lacked a mutex and the same pattern was carried through here. `AddCertsFromFile` is only called by `UseExtraCaCerts`, which is only called during Node.js startup.